arcon_state/src/backend/metrics_utils.rs

```diff
@@ -0,0 +1,13 @@
+use metrics::counter;
```

Comment:
metrics_utils file does not need cfg feature calls as mentioned by the previous comment.